pkg/katatestutils/Gopkg.toml

```diff
@@ -0,0 +1,38 @@
+# Gopkg.toml example
```

Comment:
woot! , is this correct? I thought that there should only be one `vendor` directory

Reply:
It does seem to be required, yes:

```sh
INFO: Running 'go test' as current user on package 'github.com/kata-containers/runtime/cli' with flags '-v -race -timeout 30s -outputdir "/tmp/tmp.RnuwUOlXss"'
# github.com/kata-containers/runtime/cli
package github.com/kata-containers/runtime/cli (test)
        imports github.com/kata-containers/runtime/pkg/katatestutils
        imports github.com/blang/semver: cannot find package "github.com/blang/semver" in any of:
        /home/james/go/src/github.com/kata-containers/runtime/vendor/github.com/blang/semver (vendor tree)
        /usr/local/go/src/github.com/blang/semver (from $GOROOT)
        /home/james/go/src/github.com/blang/semver (from $GOPATH)
FAIL    github.com/kata-containers/runtime/cli [setup failed]
```

... but:

```
$ dep ensure -add github.com/blang/semver
Fetching sources...

Failed to add the dependencies:

  ✗ github.com/blang/semver is already imported or required, so -add is only valid with a constraint

adding dependencies failed
$ grep -c semver Gopkg.*                
Gopkg.lock:0
Gopkg.toml:0
$
```

We do have other packages with their own vendoring, for example:

- https://github.com/kata-containers/tests/tree/master/cmd/checkcommits
- https://github.com/kata-containers/tests/tree/master/cmd/checkmetrics